integr8ly/integreatly-operator
# MGDAPI-4178 Updated test cases for RHOAM v1.23.0

# Issue link

https://issues.redhat.com/browse/MGDAPI-4179
https://issues.redhat.com/browse/MGDAPI-4180

# What


Just the update of what test cases were performed during RHOAM v1.23.0 testing.

# Verification steps




Eye review. You can check whether the test cases were indeed done as part of testing via testing epics:
https://issues.redhat.com/browse/MGDAPI-4179
